suite-native/atoms/src/Button/IconButton.tsx

```diff
@@ -1,93 +1,108 @@
-import React from 'react';
-import { TouchableOpacity } from 'react-native';
+import React, { useState } from 'react';
+import { Pressable, PressableProps } from 'react-native';
+import Animated from 'react-native-reanimated';
 
 import { Icon, IconName } from '@trezor/icons';
-import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import {
+    NativeStyleObject,
+    prepareNativeStyle,
+    mergeNativeStyles,
+    useNativeStyles,
+} from '@trezor/styles';
 
-import { ButtonColorScheme, ButtonSize } from './Button';
+import {
+    ButtonColorSchemeName,
+    ButtonSize,
+    buttonSchemeToColorsMap,
+    buttonStyle,
+    ButtonStyleProps,
+} from './Button';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { useButtonPressAnimatedStyle } from './useButtonPressAnimatedStyle';
 
-type IconButtonProps = {
+type IconButtonProps = Omit<PressableProps, 'style'> & {
```

Comment:
Same issue with `onPressIn/Out` as mention in Button

Reply:
fixed in ecbbd64b5